recipe/meta.yaml

```diff
@@ -101,8 +100,6 @@ build:
   skip: True            # [win and int(float(vc)) < 14]
 # Would like to be able to append to the calculated build string here, ping @msarahan, is this possible?
   string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}{{ linkage_nature }}{{ debug }}_cpython
-  run_exports:
-   - python_abi 3.8.* cp38
```

Comment:
Is it correct to remove run_exports @conda-forge/python?

Reply:
This needs to be added after there's a new conda-build release. There's also https://github.com/conda-forge/python-feedstock/pull/310 which uses the correct syntax for the new conda-build release.

Reply:
@isuruf

Reply:
Alright then!